styles.css

```diff
@@ -81,7 +143,37 @@ header {
 .icon > a {
   color: #505f79;
   margin-right: 12px;
-  font-size: 24px;
+  font-size: 1.5rem;
+  line-height: 1.5rem;
+}
+
+@media screen and (min-width: 768px) {
+  #headline-container {
+    background-image: url('images/headline-bg-desktop.png');
```

Comment:
> - [ ] I noticed that you used @media screen and (min-width: 768px) for every section. It's good practice to wrap all the desktop-specific code in one @media screen and (min-width: 768px) because it makes the code more organized and easier to read. By grouping all the code together, it's easier to understand what elements are specific to desktop and what elements are for mobile or other screen sizes. Additionally, it can help reduce the amount of redundant code and improve the overall performance of the website. So please consider wrapping your desktop-specific code in one @media screen and (min-width: 768px) to improve the readability and performance of your website.😊